File: Iniciante/1013.py
import math

line1 = input().split()

a = int(line1[0])
b = int(line1[1])
c = int(line1[2])

maior1 = (a+b+abs(a-b))//2
maior2 = (maior1+c+abs(maior1-c))//2

print("{} eh o maior".format(int(maior2)))

File: Iniciante/1013.js
let line1 = lines.shift().split(' ');

let a = Number(line1.shift());
let b = Number(line1.shift());
let c = Number(line1.shift());

let Maior1 = (a+b+Math.abs(a-b))/2;
let Maior2 = (Maior1+c+Math.abs(Maior1-c))/2;

console.log(Maior2 + " eh o maior");
